src/components/tab/SearchInput.vue

```diff
@@ -159,12 +168,13 @@ export default {
 				response = e.response
 			}
 			this.checkForErrorCode(response.status)
-			if (response.status === 200) await this.processWorkPackages(response.data)
+			if (response.status === 200) await this.processWorkPackages(response.data, fileId)
 			if (this.isStateLoading) this.state = STATE.OK
 		},
-		async processWorkPackages(workPackages) {
+		async processWorkPackages(workPackages, fileId) {
 			for (let workPackage of workPackages) {
 				try {
+					workPackage.fileId = fileId
```

Comment:
`this.fileInfo.id` will be the same for whole `searchInput`. you probably can use it like this without having to use it as a fn arg.
```suggestion
					workPackage.fileId = this.fileInfo.id
```

Reply:
`this.fileInfo.id` changes whenever an other file is selected in the UI, that is the basic problem. Here we want to save the fileId of the time when the search was started, so that the found workpackage will be associated with that particular fileId even when the user selects an other file (and `this.fileInfo.id` changes)